README.md

```diff
@@ -232,9 +234,9 @@ code-push release-react MyApp-Android android --targetBinaryVersion "~1.1.0"
 
 ```
 
-The CodePush client supports differential updates, so even though you are releasing your JS bundle and assets on every update, your end users will only actually download the files they need. The service handles this automatically so that you can focus on creating awesome apps and we can worry about optimizing end user downloads.
+The CodePush client supports differential updates, so even though you are releasing your JS bundle and assets on every update, you will only actually download the files you need. The service handles this optimizing downloads so that you can focus on creating awesome apps.
```

Comment:
Would keep as original. "your end users" and "they" are better words to describe the developer's users. "You" sounds more you're talking directly to the developer, which I don't think is the goal here